packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -799,6 +805,80 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         );
     }
 
+
+    /************************
+     * Experimental Opcodes *
+     ************************/
+
+    function ovmAUTH(
+        bytes32 _commit,
+        uint8 _yParity,
+        bytes32 _r,
+        bytes32 _s
+    )
+        override
+        public
+        returns (
+            address _authorized
+        )
+    {
+        // QUESTION: Should we throw here if _yParity != 0 or 1?
```

Comment:
Technically, the EIP specifies:
> if the signature is instead invalid (e.g. yParity != 0 or 1), authorized is reset to an unset value.

Reply:
doh, now seeing the next question

Reply:
kk will just reset and exit early in this case!

Reply:
`ecrecover` should fail if `yParity` is invalid, so i don't think you need to do anythin